resources/mysqlutil/mysqlutil.go

```diff
@@ -13,28 +13,50 @@ import (
 	"github.com/bytebase/bytebase/resources/utils"
 )
 
-// Instance involve the path of mysqlbinlog binary.
+type binaryName string
+
+const (
+	// MySQL is the name of mysql binary
+	MySQL binaryName = "mysql"
```

Comment:
How about `mysqlBin` and `mysqlbinlogBin`?
The camel cases are not quite meaningful.